scripts/jenkins/ardana/ansible/deploy-caasp.yml

```diff
@@ -0,0 +1,52 @@
+---
+
+- name: Deploy CaaSP using heat templates
+  hosts: hosts
+  gather_facts: False
+  vars:
+    caasp_image_url: "http://download.suse.de/install/SUSE-CaaSP-3-GM/SUSE-CaaS-Platform-3.0-for-OpenStack-Cloud.x86_64-3.0.0-GM.qcow2"
```

Comment:
we might want to cache that on clouddata instead, although d.s.d should be fast.. not a blocking issue.